x/stake/handler.go

```diff
@@ -17,18 +16,26 @@ func NewHandler(k keeper.Keeper, govKeeper gov.Keeper) sdk.Handler {
 		// NOTE msg already has validate basic run
 		switch msg := msg.(type) {
 		case types.MsgCreateValidatorProposal:
+			if sdk.IsUpgrade(sdk.BEPHHH) {
+				return sdk.ErrMsgNotSupported("MsgCreateValidatorProposal disabled in BEP-HHH").Result()
+			}
 			return handleMsgCreateValidatorAfterProposal(ctx, msg, k, govKeeper)
 		case types.MsgRemoveValidator:
+			if sdk.IsUpgrade(sdk.BEPHHH) {
+				return sdk.ErrMsgNotSupported("MsgRemoveValidator disabled in BEP-HHH").Result()
```

Comment:
As I recall about the last discussion with Vulcan, we still want to keep the ability to remove `malignant` through governance. Please do double confirm.

Reply:
Confirmed that we still need `MsgRemoveValidator`.

Reply:
fixed